### PR TITLE
allow markdown on index page, markdown=1 for article description

### PR DESCRIPTION
**Pull request by mlinksva**
Generalization of #241 by @wooorm

### CODE REVIEW

**Comment by wooorm** [comment]
Much better than my solution!

**Comment by mlinksva** [comment]
The test failure (edit link for the renamed file) is necessary, FWIW. It would go away if merged.

File: index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h1 class="alt-h1">Open Source Guides</h1>
             <h3 class="alt-h3 text-bold lh-condensed mb-2 text-black">
               {{ article.title }}
             </h3>
-            <p class="mb-0 text-gray">
+            <p markdown="1" class="mb-0 text-gray">
               {{ article.description }}
```
Comment (benbalter) [inline index.md]: Can also do `{{ article.description | markdownify }}`

Comment (mlinksva) [inline index.md]: Even better, and not necessitating any rename. Thanks!

**Comment by mlinksva** [comment]
Closing this because @benbalter's suggestion is simpler.

**Pull request closed without merging**